### ISSUE TITLE
Ruby file destroyed

### ISSUE THREAD

**Issue by steakknife** [Chiel92/vim-autoformat#247]
`EXPR_ENDEXPR_BEG                                                                                            EXPR_BEG                                                                                            EXPR_BEGEXPR_BEGEXPR_CMDARGEXPR_BEG                                                                                              EXPR_BEGEXPR_BEG                                                                                              EXPR_BEGEXPR_FNAMEEXPR_FNAMEEXPR_ENDEXPR_DOTEXPR_ENDFNEXPR_BEG                                                                                                EXPR_BEGEXPR_BEGEXPR_BEGEXPR_BEGEXPR_BEGEXPR_BEGEXPR_BEGEXPR_BEGEXPR_BEGEXPR_BEGEXPR_BEGEXPR_END|EXPR_ENDARGEXPR_END|EXPR_ENDARGEXPR_BEGEXPR_BEGEXPR_ARGEXPR_BEG                                                                                              EXPR_BEGEXPR_ENDEXPR_BEG                                                                                              EXPR_BEG                                                                                              EXPR_BEGEXPR_FNAMEEXPR_FNAMEEXPR_ENDEXPR_DOTEXPR_ENDFNEXPR_BEG                                                                                                EXPR_BEGEXPR_CMDARGEXPR_CMDARGEXPR_ARGEXPR_DOTEXPR_ARGEXPR_ARGEXPR_BEG|EXPR_LABELEXPR_BEG|EXPR_LABELEXPR_BEG|EXPR_LABELEXPR_ARGEXPR_BEG|EXPR_LABELEXPR_BEG|EXPR_LABELEXPR_CMDARGEXPR_DOTEXPR_ARGEXPR_BEG|EXPR_LABELEXPR_END|EXPR_LABELEXPR_ENDFNEXPR_DOTEXPR_ARGEXPR_ARGEXPR_ENDEXPR_DOTEXPR_ARGEXPR_BEG|EXPR_LABELEXPR_BEG|EXPR_LABELEXPR_BEG|EXPR_LABELEXPR_END|EXPR_ENDARGEXPR_ENDFNEXPR_BEG                                                                                              EXPR_BEGEXPR_ENDEXPR_BEG                                                                                            EXPR_ENDEXPR_BEG                                                                                            EXPR_BEG                                                                                            EXPR_CMDARGEXPR_DOTEXPR_ARGEXPR
`

Fantastic 👌  

Ruby 2.5.1

**Comment by steakknife** [comment]
Who are you and do you really want to start off with an unproductive finger-pointing contest?

**Comment by jeffvandyke** [comment]
Oh my lands... He owns this repository, and how about showing some appreciation and respect for someone who wrote this awesome tool, and who is volunteering his time to try and fix your issue by asking for more info on your indecipherable bug report here, all of which he is under no obligation to do? Good grief...

**Comment by steakknife** [comment]
Hahaha. This court jester is entertaining. Where do I get one of them? :) 

Anyhow, here's the MCE:

**TL;DR:** It's [a known bug](https://github.com/erniebrodeur/ruby-beautify/issues/45) in `rbeautify`, should be using https://github.com/jirutka/ruby-beautify2 because `ruby-beautify` is a dead project.

beautify2's executable `ruby-beautify` fixes this issue.

```
0 ~ ❯❯❯ echo 'class A' | rbeautify
class A
127 ~ ❯❯❯ echo 'class A = B' | rbeautify 
class A = B
127 ~ ❯❯❯ echo 'class A = B' | rbeautify
class A = B
127 ~ ❯❯❯ echo 'class A = Foo.new' | rbeautify
class A = Foo.new
127 ~ ❯❯❯ echo 'class A = Foo.new {}' | rbeautify
class A = Foo.new {}
127 ~ ❯❯❯ echo 'A = Foo.new {}' | rbeautify
EXPR_CMDARGEXPR_CMDARGEXPR_BEGEXPR_BEGEXPR_ARGEXPR_DOTEXPR_ARGEXPR_ARGEXPR_BEG|EXPR_LABELEXPR_ENDEXPR_BEG
```

### MCE

vim 8.1.300
rbeautify (0.0.7)

### Demo file `mce.rb`
```ruby
A = B
```

### `~/.vimrc`
```vim
call plug#begin()
Plug 'Chiel92/vim-autoformat'

" On-demand loading
call plug#end()

set nocompatible
filetype off
filetype plugin indent on
set wildmenu
syntax on
set modeline


" Disable annoying bells
set noerrorbells visualbell t_vb=
autocmd GUIEnter * set visualbell t_vb=
```

### Repro

```
vi mce.rb
:PluginInstall
:source ~/.vimrc
:Autoformat
```

### Result
`EXPR_CMDARGEXPR_CMDARGEXPR_BEGEXPR_BEGEXPR_ARGEXPR_BEG`

Ball in whomever's court, I don't have anymore cycles to spend on this. Moving on. ☮️
